test/plain.jl

```diff
@@ -149,6 +151,8 @@ ucoder = read(fr, "ucode")
 @assert ucode == ucoder
 salut_splitr = read(fr, "salut_split")
 @assert salut_splitr == salut_split
+salut_splitr = read(fr, "salut_2d")
+@assert salut_2d == salut_2d
```

Comment:
It's the same as every other test in this file. I didn't want to change the spirit of the testing structure.